### PR TITLE
ENH: Add epicscorelibs recipe

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hhslepicka** [comment]
This recipe depends on https://github.com/conda-forge/staged-recipes/pull/9206

**Comment by hhslepicka** [comment]
@conda-forge/help-python-c this one is ready to be reviewed/merged.
Thank you!

File: recipes/epicscorelibs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "epicscorelibs" %}
+{% set version = "7.0.2.99.0.0" %}
```
Comment (beckermr) [inline recipes/epicscorelibs/meta.yaml]: This version number is pretty long. Did you check that conda can handle it properly?

Comment (hhslepicka) [reply recipes/epicscorelibs/meta.yaml]: It is PEP440 compatible so I think it should be ok. I will look at the build log. Other than looking there, do you have other suggestions on how to test this?

Comment (hhslepicka) [reply recipes/epicscorelibs/meta.yaml]: https://github.com/mdavidsaver/epicscorelibs/blob/9a95fe4c0d20eb96a7adc8a5789e50dad75c6255/setup.py#L26-L29

Comment (hhslepicka) [reply recipes/epicscorelibs/meta.yaml]: The build log shows that conda handled it beautifully

**Pull request merged**